packages/core/demo/data/bar.ts

```diff
@@ -189,9 +188,7 @@ export const stackedHorizontalBarOptions = {
 		left: {
 			scaleType: "labels"
 		},
-		bottom: {
-			stacked: true
-		}
+		bottom: {}
```

Comment:
Yep, we deleted the flag because the stacked barchart now knows by itself he need to stack the dataset. 
We couldn't see any negative impact with that, do you find any?

Reply:
https://github.com/carbon-design-system/carbon-charts/blob/1ec4840f49262ea1422e2afd06789b7f43fd41fc/packages/core/src/charts/bar-stacked.ts#L38